### ISSUE TITLE
Cannot send direct messages

### ISSUE THREAD

**Issue by rhecl** [PLhery/node-twitter-api-v2#52]
I'm having problems with sending direct messages. As far as I saw there is no wrapper function for this feature, so I made a request as follows:

```
const client = new TwitterApi({
    appKey: consumerKey,
    appSecret: consumerSecret,
    accessToken,
    accessSecret,
});

 await client.v1.post('direct_messages/events/new', {
      event: {
        type: 'message_create',
        message_create: {
          target: {
            recipient_id: recipientId,
          },
          message_data: {
            text,
          },
        },
      },
 });
```

However, I keep getting a 403 error and I'm not sure what I'm missing.

`"name": "Error",
 "message": "Request failed with code 403"`

**Comment by alkihis** [comment]
Hi, thanks for your report.

Your request is no properly formatted. You need to add `.json` at the end of the endpoint name `direct_messages/events/new.json`.
Also, you need to add a parameter that will encode body parameters as JSON (majority of 1.1 endpoints takes only url-encoded :))

```ts
client.v1.post('direct_messages/events/new.json', { your JSON args here }, { forceBodyMode: 'json' })
```

**Comment by alkihis** [comment]
DM wrappers are now available in 1.3.0 release :) https://github.com/PLhery/node-twitter-api-v2/releases/tag/1.3.0

**Comment by rhecl** [comment]
Thanks for your help. The initial solution already worked, but I will update to the wrapper now.
